sqlmesh/utils/migrations.py

```diff
@@ -0,0 +1,13 @@
+from sqlglot.dialects.dialect import DialectType
+
+# https://dev.mysql.com/doc/refman/8.0/en/storage-requirements.html#data-types-storage-reqs-strings
+MAX_TEXT_LENGTH = 65535
```

Comment:
This seems a little extreme and will impact the page layout in mysql. Can we put something more reasonable in here? I'm sure something like 1024 or even 512 will be more than enough.

Reply:
Fair enough!

Reply:
Changed to 1024, left a comment to make it clear that it might not match MySQL's limit.